src/main/java/uk/gov/ons/census/casesvc/service/CCSPropertyListedService.java

```diff
@@ -77,7 +78,7 @@ private void handleUacQidLinksForCase(List<UacDTO> qids, Case caze) {
   }
 
   private void sendActiveCCSCaseToField(Case caze) {
-    if (!caze.isRefusalReceived() && !caze.isAddressInvalid()) {
+    if (StringUtils.isEmpty(caze.getRefusalReceived()) && !caze.isAddressInvalid()) {
```

Comment:
all these checks for non-empty strings are horrible. Just use an enum and check for non-null